mackup/applications/cyberduck.cfg

```diff
@@ -2,5 +2,8 @@
 name = Cyberduck
 
 [configuration_files]
+# based on https://trac.cyberduck.io/wiki/help/en/faq#Preferencesandapplicationsupportfileslocation
 Library/Application Support/Cyberduck
+Library/Containers/ch.sudo.cyberduck/Data/Library/Application Support/Cyberduck
+Library/Group Containers/G69SCX94XU.duck/Library/Application Support/duck
```

Comment:
I don't think this is gonna work, it's specific to each computer no?

Reply:
Yes. The `mackup` backup command works fine. i have it in my personal `.mackup` config folder. But i haven't tried running the `uninstall` and `restore` command.

But i just noticed the path in line 7 doesn't exists. If i'm not mistaken, that was for an older version of Cyberduck.

    Library/Containers/ch.sudo.cyberduck/Data/Library/Application Support/Cyberduck

But sure. i will try to split those PR a bit later. sorry for the trouble.